src/pages/home/report/ReportActionsView.tsx

```diff
@@ -87,10 +119,75 @@ function ReportActionsView({
             return;
         }
 
-        Report.openReport(reportID);
+        Report.openReport(reportID, reportActionID);
     };
 
+    useLayoutEffect(() => {
+        setCurrentReportActionID('');
+    }, [route]);
+
+    const listID = useMemo(() => {
+        isFirstLinkedActionRender.current = true;
+        const newID = generateNewRandomInt(listOldID, 1, Number.MAX_SAFE_INTEGER);
+        listOldID = newID;
+        return newID;
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [route, isLoadingInitialReportActions]);
+
+    const indexOfLinkedAction = useMemo(() => {
+        if (!reportActionID || isLoading) {
+            return -1;
+        }
+
+        return allReportActions.findIndex((obj) => String(obj.reportActionID) === String(isFirstLinkedActionRender.current ? reportActionID : currentReportActionID));
+    }, [allReportActions, currentReportActionID, reportActionID, isLoading]);
+
+    const reportActions = useMemo(() => {
+        if (!reportActionID) {
+            return allReportActions;
+        }
+        if (isLoading || indexOfLinkedAction === -1) {
+            return [];
+        }
+
+        if (isFirstLinkedActionRender.current) {
+            return allReportActions.slice(indexOfLinkedAction);
+        }
```

Comment:
`isFirstLinkedActionRender.current` is true only for the first render when we position the linked message. We slice it from the older messages to the linked message so that the linked message is located at the bottom. Then, fetchNewer is called, and we expand this with new items.